src/API.ts

```diff
@@ -82,23 +94,37 @@ const skypeTokenAuth = async (chatToken: OmnichannelChatToken): Promise<Response
     }
 }
 
+
+const getTypeFromFile = (type: string, name: string, operation: string) => {
+
+    if (type.includes('image')) {
+        const stripFileName = name.split('.');

```

Comment:
Can we use `name.substring(name.lastIndexOf(".") + 1, name.length)` instead? It would cover the scenario where a file name containing multiple `"."`.

Reply:
Ill do.

Reply:
yeah, I see your point..., I'll update it,